general/releases.rst

```diff
@@ -28,11 +28,11 @@ old, upcoming or future release.
 +---------------+--------------+
 | OP-TEE 3.16.0 | 14/Jan/22    |
 +---------------+--------------+
-| OP-TEE 3.15.0 | 15/Oct/21    |
```

Comment:
I have a public holiday on earlier planned date October 15th. So will send out the rc1 tag as planned on Oct 8th but make the final release on Oct 18th instead of Oct 15th. I hope that won;t cause any issues.